### PR TITLE
DOC: Make code block an Item List

### LINKED ISSUE #14277: Broken reference in docstring of scipy.stats.power_divergence
Quick Link: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.power_divergence.html#scipy.stats.power_divergence

The parameter `lambda_` creates a reference to a paper in its documentation, but this reference is not rendered in the docs. Instead, a mangled string is shown. This happens twice for different references; however, the raw docstring looks good.

Not sure if this is caused by something in scipy, if it could be a bug in sphinx, or something else entirely. If anybody knows the cause, I can submit a PR once a solution has been identified.

### PR DESCRIPTION
**Pull request by FirefoxMetzger**
Closes: https://github.com/scipy/scipy/issues/14277

The issue was that references were not rendered correctly. I decided to replace the code block with a definition list in response to @tupui 's comment https://github.com/scipy/scipy/issues/14277#issuecomment-867134453 .

### CODE REVIEW

**Comment by FirefoxMetzger** [comment]
I just noticed that the CI doesn't seem to build any docs with the PR (perhaps I missed it).

What is the preferred way to verify that the docs are getting rendered correctly?

**Comment by tylerjereddy** [comment]
Something of a guess, but your SciPy fork `master` branch is almost 800 commits behind the main repo `master`, and CircleCI tends to test the PR branch directly without merging.

So, my first suggestion is to rebase this feature branch on the latest SciPy `master`.

**Comment by tylerjereddy** [comment]
Actually, nevermind, you're only 13 commits behind for this branch I think.

**Comment by tylerjereddy** [comment]
I'll try to close/re-open.

**Comment by FirefoxMetzger** [comment]
Strange; I did the edit directly in GH on the master branch (since it is so small).

Thanks for the heads up @tylerjereddy I'll investigate :)

File: scipy/stats/stats.py [hunk 1]
```diff
@@ -6574,18 +6574,21 @@ def power_divergence(f_obs, f_exp=None, ddof=0, axis=0, lambda_=None):
...
-                                        the G-test [3]_.
-            "freeman-tukey"      -1/2   Freeman-Tukey statistic.
-            "mod-log-likelihood" -1     Modified log-likelihood ratio.
-            "neyman"             -2     Neyman's statistic.
-            "cressie-read"        2/3   The power recommended in [5]_.
+        strings, in which case the corresponding numerical value is used
+        
+        Pearson (value 1)
+            Pearson's chi-squared statistic. In this case, the function is
+            equivalent to `stats.chisquare`.
```
Comment (peterbell10) [inline scipy/stats/stats.py:6581]:
```suggestion
            equivalent to `chisquare`.
```

Should fix the doc error: `WARNING: py:obj reference target not found: stats.chisquare`

**Comment by FirefoxMetzger** [comment]
@tylerjereddy was right. Turns out I had an old scipy fork which GH used as a base when I created the edit from the scipy/scipy repo. I fast-forwarded all the scipy commits I was missing.

Now, two tests run into a 1h timeout (is this a known issue, or something I may have introduced in the fast-forward merge?); and I again don't get the circle CI docs :(

**Comment by FirefoxMetzger** [comment]
I still don't seem to be getting CircleCI doc builds with this...

@tupui Do you think it makes sense for me to close this PR and re-open it from a new branch? Or do doc builds need to be triggered manually?

**Comment by tupui** [comment]
> I still don't seem to be getting CircleCI doc builds with this...
> 
> @tupui Do you think it makes sense for me to close this PR and re-open it from a new branch? Or do doc builds need to be triggered manually?

No don't do another PR. I would just rebase on master here.

**Comment by FirefoxMetzger** [comment]
@tupui Done. The git history of this branch is probably a bit of a mess now, so I would be very happy if this branch can be squashed before the merge :)

In any case, the CircleCI test is there now, so I hope that we can finally proceed with this xD For a simple doc PR this has been idling quite a while.

File: scipy/stats/stats.py [hunk 2]
```diff
@@ -6574,18 +6574,21 @@ def power_divergence(f_obs, f_exp=None, ddof=0, axis=0, lambda_=None):
...
-                                        equivalent to `stats.chisquare`.
-            "log-likelihood"      0     Log-likelihood ratio. Also known as
-                                        the G-test [3]_.
-            "freeman-tukey"      -1/2   Freeman-Tukey statistic.
-            "mod-log-likelihood" -1     Modified log-likelihood ratio.
-            "neyman"             -2     Neyman's statistic.
-            "cressie-read"        2/3   The power recommended in [5]_.
+        strings, in which case the corresponding numerical value is used
+
+        Pearson (value 1)
```
Comment (tupui) [inline scipy/stats/stats.py:6579]: The previous version was better in a sense that you should do `lambda_="pearson"` and not `lambda_="Pearson"`. 

Here you should do this instead (checkout `scipy.stats.kstest` for example with `mode` for instance)

```
* 'pearson', 1: Pearson's chi-squared...
```
or
```
* ``pearson``, 1: Pearson's chi-squared...
```

**Review by tupui — CHANGES_REQUESTED** [review]
Thanks for the update @FirefoxMetzger. It looks better with the rebase. Don't worry about the history, before merging we always look before merging and decide on what we think is the best strategy.

File: scipy/stats/stats.py (hunk 2 above)
Comment (FirefoxMetzger) [reply scipy/stats/stats.py:6579]: ah so it's a basic list instead of a definition list. Can do.

Comment (FirefoxMetzger) [reply scipy/stats/stats.py]: @tupui Done :)

File: scipy/stats/stats.py [hunk 3]
```diff
@@ -6574,18 +6574,21 @@ def power_divergence(f_obs, f_exp=None, ddof=0, axis=0, lambda_=None):
...
-                                        equivalent to `stats.chisquare`.
-            "log-likelihood"      0     Log-likelihood ratio. Also known as
-                                        the G-test [3]_.
-            "freeman-tukey"      -1/2   Freeman-Tukey statistic.
-            "mod-log-likelihood" -1     Modified log-likelihood ratio.
-            "neyman"             -2     Neyman's statistic.
-            "cressie-read"        2/3   The power recommended in [5]_.
+        strings, in which case the corresponding numerical value is used:
+
+        * ``pearson`` (value 1)
```
Comment (tupui) [inline scipy/stats/stats.py:6579]:
```suggestion
        * ``"pearson"`` (value 1)
```

**Review by tupui — APPROVED** [review]
@FirefoxMetzger LGTM with the typo. Thanks!
@peterbell10 fine with you? I will squash merge this tomorrow.

**Pull request merged**